game/src/main/scala/io/aigar/game/GameThread.scala

```diff
@@ -54,6 +66,18 @@ class GameThread(scoreThread: ScoreThread, playerIDs: List[Int]) extends Runnabl
   }
 
   def updateGames: Unit = {
+    // Test to see if the ranked is over
```

Comment:
Do you think we could move this to a separate method rather than adding a comment? Naming ideas: `resetRankedGameIfExpired`, `resetRankedGameIfOver`, `updateRankedGameExpiry`, etc.